roles/custom/matrix-bridge-hookshot/templates/config.yml.j2

```diff
@@ -107,6 +107,14 @@ metrics:
   # (Optional) Prometheus metrics support
   #
   enabled: {{ matrix_hookshot_metrics_enabled | to_json }}
+{% if matrix_hookshot_experimental_encryption_enabled %}
+queue:
+  monolithic: true
+  port: 6379
+  host: matrix-redis
```

Comment:
These assumptions should not be hardcoded and should use variables (`matrix_hookshot_queue_host` and `matrix_hookshot_queue_port`).

The port could default to `6379`, since that's quite standard for Redis.

The host should default to empty. Perhaps you can only enable the queue (`if` statement) if the host is non-empty. Example: `{% if matrix_hookshot_queue_host %}`

The `matrix_hookshot_queue_host` and `matrix_hookshot_queue_port` variables can be set to the correct values (`matrix-redis`, ..) via `group_vars/matrix_servers` - the file we use for wiring the different roles together. It's better to explicitly wire things via this central file, instead of having roles randomly assume things about other roles or each into each other's variables.

`matrix_hookshot_queue_host` would be defined as such in `group_vars/matrix_servers`: 

```yml
matrix_hookshot_queue_host: "{{ redis_identifier if matrix_hookshot_experimental_encryption_enabled and redis_enabled else '' }}"
```

This: provides sane defaults for these variables, potentially overrides them via the group vars file.. and still allows people to use external Redis instances, if they want to.